jerry-debugger/src/lib/cdt-controller.ts

```diff
@@ -67,12 +70,68 @@ export class CDTController {
     }
   }
 
-  requestBreakpoint() {
+  sendPaused(breakpoint: Breakpoint) {
     if (!this.protocolHandler || !this.proxyServer) {
       throw new Error('missing object dependencies');
     }
-    this.protocolHandler.getLastBreakpoint();
+
     // Node uses 'Break on start' but this is not allowable in crdp.d.ts
-    this.proxyServer.sendPaused('other');
+    this.proxyServer.sendPaused(breakpoint, 'other');
+  }
+
+  requestBreakpoint() {
+    if (!this.protocolHandler) {
+      throw new Error('missing protocol handler');
+    }
+    const breakpoint = this.protocolHandler.getLastBreakpoint();
+    if (!breakpoint) {
+      throw new Error('no last breakpoint found');
+    }
+
+    this.sendPaused(breakpoint);
+  }
+
+  requestStepOver() {
+    if (!this.protocolHandler) {
+      throw new Error('missing protocol handler');
+    }
+    this.protocolHandler.stepOver();
```

Comment:
Yeah, I'd use `this.protocolHandler!.stepOver();` and remove the `if(!this.protocolHandler) {` checks.

Reply:
OK I'll make a follow-on patch to reduce all this error cruft.

Reply:
Or perhaps a private `getProtocolHandler()` method that does the error check in one place?